### DISCUSSION TITLE
Hair mods crashing the game

### DISCUSSION THREAD

**Discussion by mmbrubs** [xivdev/Penumbra#148] [Q&A]
So, everytime i try using a hair mod and my game just closes (or the mod doesnt work, making my character go bald) and i have to disable them on Penumbra before clicking to play the game again. I don't know if the picture can help but it looks like this. I already tried to disable all my mods but it still crashes. 
![image](https://user-images.githubusercontent.com/99089353/152650546-068c0fae-6dd8-4416-a4a9-0ec992c35359.png)

**Comment by Ottermandias** [comment]
I'm extremely surprised that anything works with that non-ascii symbol in your path.
